packages/jack_link/PKGBUILD

```diff
@@ -0,0 +1,36 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+_link_commit="41d9aa111f702e78b6fbaee9d3e06dda1db6420d"
+pkgname=jack_link
+pkgver=0.2.3
+pkgrel=1
+pkgdesc='JACK transport bridge to Ableton Link'
+arch=(aarch64 x86_64)
+url='https://github.com/rncbc/jack_link'
+license=(GPL-2.0-or-later)
+depends=(gcc-libs glibc)
+makedepends=(asio jack)
+groups=(pro-audio)
+source=("$pkgname-$pkgver.tar.gz::https://github.com/rncbc/jack_link/archive/refs/tags/v$pkgver.tar.gz"
+        "link-$_link_commit.tar.gz::https://github.com/Ableton/link/archive/$_link_commit.tar.gz")
```

Comment:
I personally found that since pacman supports checksums for VCS sources it's slightly easier to maintain packages using a git source and the submodule head being checked out at the correct commit automatically, see e.g. stomptuner or vcvrack.

Reply:
That then also requires the main project sources to be a git URL, because you then need the `.gitmodules` file.

I like the state of sub-module projects to be a little more explicit in the PKGBUILD.